html/pfappserver/lib/pfappserver/I18N/i_default.po

```diff
@@ -5935,6 +5935,10 @@ msgid "pf::profile::filter::connection_type"
 msgstr "Connection Type"
 
 # pf::filter (Portal Profile Filters)
+msgid "pf::profile::filter::day_of_week"
+msgstr "Day Of Week"
```

Comment:
Any special reason why it shouldn't be written 'Day of week' (without capital letters at the beginning of each word)